### DISCUSSION TITLE
Why are some channels/countries not available on https://iptv-org.github.io/ ?

### DISCUSSION THREAD

**Discussion by Nod3820** [iptv-org/community#569] [Q&A] [answered]
Why are some channels/countries not available on https://iptv-org.github.io/ ?

I was searching for some sports channels but I saw that I can't click on some channels or countries, I don't know why, and it will be cool if all channels and countries was like free to see, so, I will going to ask 2 questions:

- Why are some channels/countries not available on https://iptv-org.github.io/ ?
- Is it possible to make possible all channels and countries available?

Thanks for reading and answer
![sufian](https://github.com/iptv-org/iptv-org.github.io/assets/121915817/e167ced0-1360-4e99-8cf8-e1fd21ba14f5)
![sufgian2](https://github.com/iptv-org/iptv-org.github.io/assets/121915817/e472fa6f-2843-49c4-a3ff-adbd3075008d)

**Comment by Dum4G** [comment, marked as answer]
https://iptv-org.github.io/ is a database that serves a purpose of storing info about channels description and EPG guides available for them. The fact that we KNOW about them doesn't always mean that we have a working stream unfortunately. Sometimes, there are countries that have no available channels for them for the time being

**Reply by Nod3820** [reply]
ok, sorry for asking that question

**Reply by Dum4G** [reply]
No worries

**Comment by jaketheb8** [comment]
> https://iptv-org.github.io/ is a database that serves a purpose of storing info about channels description and EPG guides available for them. The fact that we KNOW about them doesn't always mean that we have a working stream unfortunately. Sometimes, there are countries that have no available channels for them for the time being

Is this also why some channels just don't play anything when you click on them? I either get an error sign, a no wi-fi sign, or endless loading loop

**Reply by Dum4G** [reply]
What service/player do you use? We contain info about channel existance and it's stream sources separately in different repos. Unfortunately, we can't guarantee them to be everlasting and always available across the globe

**Reply by jaketheb8** [reply]
I use Open IPTV on my Xbox and a few apps on my phone, but I totally get the issues of trying to broadcast especially obscure stations everywhere, hopefully one day it'll be easier
